frontend/__tests__/IncomingReviewsModal.test.js

```diff
@@ -79,11 +79,19 @@ describe('IncomingReviewsModal', () => {
     postRequest.mockImplementation(mockPostRequest)
     mountModal()
 
+    const emptyInboxMessage = wrapper.findComponent({
+      ref: 'emptyInboxMessage',
+    })
+    expect(emptyInboxMessage.exists()).toBeTruthy()
+    expect(emptyInboxMessage.vm.$el).toBeTruthy()
+
     wrapper.vm.reviews = [
       { id: 0, reviewer: { first_name: 'first', last_name: 'last' } },
     ]
-
     expect(wrapper.vm.reviews.length).toBe(1)
+
+    await wrapper.vm.$nextTick()
+    expect(emptyInboxMessage.exists()).toBeFalsy()
```

Comment:
Do we need something like this in this test too? (Not sure the difference between `el` or just using `exists`) `expect(emptyInboxMessage.$el).toBeFalsy()`

Reply:
That check is omitted in those nonexistence tests because a wrapper for an element that doesn't exist has no `vm` property, so trying to access `vm.$el` on such an element throws an error.

(btw checking whether `vm.$el` is truthy checks whether the component is mounted, basically an extra check to make sure that the element is visible)